src/app/models/form.server.model.ts

```diff
@@ -184,22 +184,24 @@ EncryptedFormDocumentSchema.methods.addPaymentAccountId = async function ({
   accountId: FormPaymentsChannel['target_account_id']
   publishableKey: FormPaymentsChannel['publishable_key']
 }) {
-  this.payments_channel = merge(this.payments_channel, {
-    target_account_id: accountId,
-    publishable_key: publishableKey,
-    enabled: true,
-  })
+  if (!this.payments_channel) {
+    this.payments_channel = {
+      // Definitely Stripe for now, may be different later on.
+      channel: PaymentChannel.Stripe,
+      target_account_id: accountId,
+      publishable_key: publishableKey,
+    }
+  }
```

Comment:
cmiiw, `this.payments_channel.enabled` is different from `this.payments_field.enabled`, and `enabled: true` isn't supposed to be here as it's not in the schema/not needed either.

Reply:
yup, that's right! `enabled` doesn't exist on the `payment_channel` object type